app/Http/Controllers/Transfer/Bank/WithdrawalController.php

```diff
@@ -16,12 +22,79 @@ class WithdrawalController extends Controller
      *
      * @param PayPalServiceProvider $paypal
      * @param Request $request
-     * @return Payment
+     * @return \PayPal\Api\PayoutBatch
      */
 
     public function withdraw(PayPalServiceProvider $paypal, Request $request)
     {
+        /**
+         * This is the output
+         * {
+                    "sender_batch_header":{
+                        "sender_batch_id":"2014021801",
+                        "email_subject":"You have a Payout!"
+                    },
+                    "items":[
+                        {
+                            "recipient_type":"EMAIL",
+                            "amount":{
+                                "value":"1.0",
+                                "currency":"USD"
+                            },
+                            "note":"Thanks for your patronage!",
+                            "sender_item_id":"2014031400023",
+                            "receiver":"shirt-supplier-one@mail.com"
+                        }
+                    ]
+                }
+         */
+
```

Comment:
There will be a need for internal checking of wallets before payouts is executed.